src/components/ExtraPages/Help/DataHelp.js

```diff
@@ -1,35 +1,154 @@
 import React from "react";
-import { Container, Image } from "react-bootstrap";
-import { DataInfoContents, DataInfoHead } from "./Contents/DataInfo";
+import { Container, Image, Row, Col } from "react-bootstrap";
+import HelpNav from "../HelpNav";
+
 import "../ExtraPages.css";
 
 export default function DataHelp() {
   return (
     <Container className="help">
-      <div>
-        <h1>{DataInfoHead.title}</h1>
-        <p>
-          {DataInfoHead.intro[0]}
-          <a href="/indices-help">{DataInfoHead.intro[1]}</a>
-        </p>
-      </div>
-      <br></br>
-      {/* section about the data */}
-      {DataInfoContents.map((data, i) => (
-        <div key={i}>
-          <h5>{data.source}</h5>
-          <p>
-            {data.description}{" "}
-            {data.link && <a href={data.link}>More info here</a>}
-          </p>
-          <Image
-            className="mb-3"
-            fluid="true"
-            src={`${process.env.PUBLIC_URL}${data.image}`}
-            alt=""
-          />
-        </div>
-      ))}
+      <Row fluid="true">
+        {/* sidenav for help pages */}
+        <Col
+          xs={3}
+          sm={2}
+          m={1}
+          style={{ paddingLeft: 0, marginLeft: "-10%", marginRight: "10%" }}>
+          <HelpNav />
+        </Col>
+        {/* page contents */}
+        <Col>
+          <div>
+            <h1>Information on Data Sources</h1>
+            <p>
+              The main sources of climate information in this plot explorer are
+              the EuroCORDEX and UKCP18 regional ensembles, but several other
+              sources are also used: brief descriptions of these are provided
+              here. All indices are first computed on each model's native grid,
+              then{" "}
+              <a href="/indices-help">
+                regridded to a common 12km OSGB grid to facilitate comparisons.
+              </a>
+            </p>
+          </div>
+          <br></br>
+          {/* section about the data */}
+          <div id="cmip5-13">
+            <h5>CMIP5-13 ensemble</h5>
+            <p>
+              This consists of simulations from thirteen of the coupled
+              ocean-atmosphere models run as part of the Coupled Model
```

Comment:
I can't make a commit suggestion because it's too lines:

```html
{" "}<a href="https://www.wcrp-climate.org/wgcm-cmip/wgcm-cmip5"> 
    Coupled Model Intercomparison Project Phase 5 (CMIP5)
</a>{" "}